android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/DotsIndicator.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun DotsIndicator(
+    totalDots: Int,
+    selectedIndex: Int
+) {
+    LazyRow(
+        modifier = Modifier
+            .wrapContentWidth()
+            .wrapContentHeight()
+    ) {
+        items(totalDots) { index ->
+            if (index == selectedIndex) {
+                Box(
+                    modifier = Modifier
+                        .size(8.dp)
+                        .clip(CircleShape)
+                        .background(Color.White)
+                )
+            } else {
+                Box(
+                    modifier = Modifier
+                        .size(8.dp)
+                        .clip(CircleShape)
+                        .background(Color.White.copy(alpha = 0.2f))
+                )
+            }
```

Comment:
I'd suggest defining the modifier outside the block for reuse purposes. Could you please try this?

```
            val boxModifier = Modifier
                        .size(8.dp)
                        .clip(CircleShape)
            if (index == selectedIndex) {
                Box(
                    modifier = boxModifier
                         .background(Color.White)
                )
            } else {
                Box(
                    modifier = boxModifier
                        .background(Color.White.copy(alpha = 0.2f))
                )
            }
```

Reply:
Updated in https://github.com/suho/kmm-ic/pull/106/commits/292662b6eebc6d1b2d93c12abc3ad4e6f6ecc4b7

Reply:
@suho About this update, I think we could refactor a bit more to get rid of redundant lines in here

```suggestion
            val backgroundColor = if (index == selectedIndex) {
                Color.White
            } else {
                Color.White.copy(alpha = 0.2f)
            }
             Box(
                    modifier = Modifier
                        .size(8.dp)
                        .clip(CircleShape)
                        .background(backgroundColor)
             )

```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/106/commits/029cfc14c4b123b651f9b3115660969719838097